src/Serilog.Sinks.Seq/Sinks/Seq/Batched/BatchedSeqSink.cs

```diff
@@ -60,11 +59,11 @@ public async Task OnEmptyBatchAsync()
         if (_controlledSwitch.IsActive &&
             _nextRequiredLevelCheckUtc < DateTime.UtcNow)
         {
-            await EmitBatchAsync(Enumerable.Empty<LogEvent>());
+            await EmitBatchAsync([]);
         }
     }
 
-    public async Task EmitBatchAsync(IEnumerable<LogEvent> events)
+    public async Task EmitBatchAsync(IReadOnlyCollection<LogEvent> events)
```

Comment:
Ah I see that's a pretty significant API difference between the old batching sink and the new one? `IReadOnlyCollection` does seem like a better interface here.